docs/01-getting-started.md

```diff
@@ -22,6 +22,42 @@ cannot be run from the `sleeper` directory directly as the Java CDK classes are
 `cdk.json`.  See [this link](https://docs.aws.amazon.com/cdk/latest/guide/bootstrapping.html) for guidance on how to 
 bootstrap CDK in your account.
 
+### Deployment environment
+
+You can use the CDK to create an EC2 instance in a VPC that is suitable for deploying Sleeper. Once the CLI is logged in
+to your AWS, and the CDK has been bootstrapped, run these commands in the directory `java/cdk-environment` to build and
+deploy the environment:
+
+```bash
+mvn clean install
+cdk deploy
+aws cloudformation describe-stacks --query "Stacks[*].Outputs[?OutputKey=='ConnectCommand'].OutputValue" --output text
+```
+
+This will create an SSH key locally, and output a command to connect to the EC2 instance with that key. It should look
+like this:
+
+```bash
+ssh -i BuildEC2.pem ubuntu@[ec2-public-ip]
+```
+
+Immediately after it's deployed, commands will run on this instance to install development tools. Once you're connected,
+you can check the progress of those commands like this:
+
+```bash
+cloud-init status
+```
+
+You can check the output like this (add `-f` if you'd like to follow the progress):
+
+```bash
+tail /var/log/cloud-init-output.log
+```
+
+Once it's finished the instance might restart. The Sleeper Git repository will be checked out at `~/sleeper`.
+
```

Comment:
Can you add a note on how to tear the whole EC2/VPC stack down?

Reply:
Done